apps/portals/src/portal-components/ProjectDiscussionForum.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react'
+import { useGetEntityForum } from 'synapse-react-client/dist/utils/hooks/SynapseAPI/entity/useGetEntityForum'
+import { DiscussionThread } from 'synapse-react-client/dist/containers/discussion_forum/DiscussionThread'
+import { ForumPage } from 'synapse-react-client/dist/containers/discussion_forum/ForumPage'
+import ArrowBackIcon from '@mui/icons-material/ArrowBack'
+import { useHistory, useLocation } from 'react-router-dom'
+import { Box, IconButton } from '@mui/material'
+
+const ProjectDiscussionForum = () => {
```

Comment:
Wrap the ForumPage and DiscussionThread components from SRC, using a search parameter to determine which one should be shown.  Introduced the idea of a search parameter that does not have impact on the query filters of a DetailsPage (suffix "NotAQueryFilterKey"!).  Let me know what you think about this approach